apps/communications/contacts/style/matching_contacts.css

```diff
@@ -114,6 +113,16 @@ form[role="dialog"][data-type="edit"] #main {
   border-style: solid;
 }
 
+/* Ellipsis direction hack before bug 883884 is fixed */
+.ellipsis-dir-fix {
```

Comment:
We widely use this rule in SMS, so I thought it would be useful for Contacts as well :)